spec/features/user_creates_moment_spec.rb

```diff
@@ -0,0 +1,94 @@
+describe "UserCreatesAMoment", js: true do
+  let(:user) { create :user2, :with_allies }
+  let!(:category) { create :category, userid: user.id }
+  let!(:mood) { create :mood, userid: user.id }
+  let!(:strategy) { create :strategy, userid: user.id }
+
+  feature 'Creating, viewing, and editing a moment' do
+    it 'is successful' do
+      login_as user
+      visit moments_path
+
+      within '#page_title_content' do
+        expect(page).to have_content 'Moments'
+      end
+
+      expect(page).to have_content 'Delve deep into your moments - events and situations that affect your mental health.'
```

Comment:
Well.. we could.. other feature specs assume everything is in English too so if we're going to take that stance, it would be nice to apply it to all feature specs since there aren't that many right now. I think it's a trade-off, using the key means that if you change the translation, you don't have to update the test, but it also makes the tests a little less readable so it adds time to one's development cycle.